dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/EventExporterTest.java

```diff
@@ -810,6 +810,26 @@ void testOrderByOccurredAtAsc()
         assertEquals( List.of( "nxP7UnKhomJ", "TvctPPhpD8z" ), enrollments );
     }
 
+    @Test
+    void shouldSortEntitiesRespectingOrderWhenOrderAndAttributeOrderSupplied()
```

Comment:
Nice that you were able to find a test that is failing on master and shows the issue 😄